manifests/init.pp

```diff
@@ -39,6 +39,12 @@
 # @param postgresql_db_password
 #   Password of Pulp database
 #
+# @param django_secret_key
+#   SECRET_KEY for Django
+#
+# @param redis_db_number
```

Comment:
I'd drop the `_number` part since a DB is always a number in Redis. With the datatype this is also enforced.

Reply:
Point taken and done.